src/Simplex/Chat/Controller.hs

```diff
@@ -967,6 +1017,21 @@ instance ToJSON SQLiteError where
 throwDBError :: ChatMonad m => DatabaseError -> m ()
 throwDBError = throwError . ChatErrorDatabase
 
+data RemoteError
```

Comment:
```suggestion
data RemoteHostError
```

Reply:
The errors appear to be the same for the controller and host.